tests/src/JIT/Regression/JitBlue/GitHub_3449/GitHub_3449.cs

```diff
@@ -15,26 +15,19 @@ public class Program
     // 43e08b2a2c280291, which is different from what the codegen produces. To fix this inconsistency,
     // the constant folding is forced to have the same behavior as the codegen, and the result
     // must be always 43e08b2a2c280290.
+    // 
+    // We don't have a good way to tell if the CLR is compiled by clang or VC++, so we simply allow
+    // both answers.
+
```

Comment:
@CarolEidt I agree it is confusing. As such, I don't have a good idea for a fix. Do you have a proposed wording that you would like?